lib/Decompressor.js

```diff
@@ -63,12 +92,8 @@ export class Decompressor extends Transformer {
    */
   async decompress({compressedBytes, diagnose} = {}) {
     this.contextMap = new Map();
-    this.termToId = new Map(KEYWORDS);
     this.nextTermId = FIRST_CUSTOM_TERM_ID;
```

Comment:
I would expect `this.nextTermId = FIRST_CUSTOM_TERM_ID;` to be in `this._reset()`.

... oh and it is! We can remove this other line then.

```suggestion
```